src/index.js

```diff
@@ -60,6 +60,23 @@ class PaperWallet {
     return { claimData: claimData, qr: qr, version: ver }
   }
 
+  /**
+   * creates the wallet
+   * @param {any} vc
+   * @param {any} resultLink
+   */
+  async createWalletVc (vc, resultLink) {
```

Comment:
Since we only use the claimdata of the vc, we might as well take that as a parameter.